internal/services/resource/resource_group_resource.go

```diff
@@ -131,7 +131,7 @@ func resourceResourceGroupDelete(d *pluginsdk.ResourceData, meta interface{}) er
 	if meta.(*clients.Client).Features.ResourceGroup.PreventDeletionIfContainsResources {
 		resourceClient := meta.(*clients.Client).Resource.ResourcesClient
 		// Resource groups sometimes hold on to resource information after the resources have been deleted. We'll retry this check to account for that eventual consistency.
-		err = pluginsdk.Retry(2*time.Minute, func() *pluginsdk.RetryError {
+		err = pluginsdk.Retry(7*time.Minute, func() *pluginsdk.RetryError {
```

Comment:
Would 5 already be sufficient?

Reply:
I'm not sure! I thought 2 was until it wasn't. It seems random how long a resource group will hang on to resources that have been deleted. We can give 5 a shot and see how that goes